web_app/models/atmospheric_corrosion_models.py

```diff
@@ -5,6 +5,7 @@
 class atmospheric_corrosion_model:
```

Comment:
Should be one model base class since the functions are the same as the Immersion corrosion model class

Reply:
changed to one parent class